.mypy.ini

```diff
@@ -0,0 +1,12 @@
+[mypy-Bio/PDB/*]
```

Comment:
What do you think about adding the following configuration? Most of these are to make the displayed errors easier to understand :) and all is documented [here](https://mypy.readthedocs.io/en/stable/config_file.html).

@peterjc do you also have any preferences here?

```
[mypy]
show_error_context = true
show_column_numbers = true
show_error_codes = true
pretty = true
warn_redundant_casts = true
```

Reply:
No comment: I am currently unfamiliar with these settings

Reply:
Works for me, thanks!